### PR TITLE
Fixed response code and range header for partial content of getItems in REST API

### PR DESCRIPTION
**Pull request by edgardmessias**

### CODE REVIEW

File: inc/apirest.class.php [hunk 1]
```diff
@@ -203,10 +203,18 @@ public function call() {
                   $response = $this->getItems($itemtype, $this->parameters, $totalcount);
 
                   //add pagination headers
-                  if (!isset($this->parameters['range'])) {
-                     $this->parameters['range'] = "0-50";
+                  $range = [0,50];
```
Comment (orthagh) [inline inc/apirest.class.php]: same

File: inc/api.class.php [hunk 2]
```diff
@@ -919,9 +919,12 @@ protected function getItems($itemtype, $params = array(), &$totalcount=0) {
             $range = explode("-", $params['range']);
             $params['start']      = $range[0];
             $params['list_limit'] = $range[1]-$range[0]+1;
+            $params['range']      = $range;
          } else {
             $this->returnError("range must be in format : [start-end] with integers");
          }
+      } else{
+         $params['range'] = array(0,50);
```
Comment (orthagh) [inline inc/api.class.php]: instead 50, $_SESSION['glpilist_limit'] could be better. It can be set and controlled in the interface

**Comment by edgardmessias** [comment]
ok, I can make this changes

**Comment by orthagh** [comment]
Thank you for this contribution

**Pull request merged**